sections/semantics-grouping-content.include

```diff
@@ -1400,7 +1398,7 @@
     meaning.
   </div>
 
-  The <span class="impl">first</span> <{figcaption}> element child of the element, if any,
+  Any <{figcaption}> element child of <{figure}>, if any,
```

Comment:
Please change "element child" to "descendants". ", if any" can be removed - it is redundant with the start of the sentence.